test/test_utils.hpp

```diff
@@ -39,14 +37,58 @@ struct LayerParams
 
 //****************************************************************************
 template <class T>
-inline bool almost_equal(T v1, T v2, float rtol = 1e-02, float atol = 1e-06)
+inline bool almost_equal(T v1, T v2, float rtol = 5e-03, float atol = 1e-05)
```

Comment:
What is the rationale for lowering the tolerance? Generally, tolerance for single precision float is around 1e-6 to 1e-7.

Reply:
I don't fully grok the new almost_equal() and we should discuss.  I have tried to do some reading on this and I thought that that would correspond to rtol being lowered 1e-6 or 1e-7.   We have some answers that are varying by slightly less than 1% which what I think rtol is capturing....I was trying to make the test more sensitive and maybe this was a mistake.

Reply:
This is something to discuss in Tuesday meetings